cg/services/order_validation_service/workflows/fastq/models/sample.py

```diff
@@ -0,0 +1,14 @@
+from cg.models.orders.sample_base import PriorityEnum, SexEnum
+from cg.services.order_validation_service.constants import ElutionBuffer
+from cg.services.order_validation_service.models.sample import Sample
+
+
+class FastqSample(Sample):
```

Comment:
We seem to be missing SubjectId (seems to be missing in Atlas as well).

Reply:
Should that be in the parent class instead?

Reply:
Nope, a subject id refers to a person, and for our nonhuman order types, e.g. mutant, you will not be able to send a subject id

Reply:
fixed